packages/desktop/views/dashboard/drawers/dapp-config/components/ConnectionSummary.svelte

```diff
@@ -0,0 +1,117 @@
+            })
+
+            const isEnabled = persistedNamespaces.some((namespace) => {
+                return supportedMethods.some((method) => namespace.methods.includes(method))
+            })
+
+            permissions.push({ label: permission, enabled: isEnabled, required: isRequired })
+        }
+
+        permissionPreferences = permissions
+    }
+
+    let networkPreferences: string[] = []
+    function setNetworkPreferences(): void {
+        const networks = []
+        for (const namespace of Object.values(persistedDappNamespace)) {
+            for (const chain of namespace.chains) {
+                networks.push(chain)
+            }
+        }
+        networkPreferences = Object.values(networks)
+    }
+
+    let accountPreferences: IAccountState[] = []
+    function setAccountPreferences(): void {
+        const accounts = {}
+        for (const namespace of Object.values(persistedDappNamespace)) {
+            for (const accountWithNetworkId of namespace.accounts) {
+                const [namespace, chainId, address] = accountWithNetworkId.split(':')
+                const account = findActiveAccountWithAddress(address, `${namespace}:${chainId}` as NetworkId)
+                if (account) {
+                    accounts[account.index] = account
+                }
+            }
+        }
+        accountPreferences = Object.values(accounts)
+    }
+
+    onMount(() => {
+        setPermissionPreferences()
+        setNetworkPreferences()
+        setAccountPreferences()
+    })
+</script>
+
+<selection-component class="flex flex-col gap-4">
+    <Text textColor="secondary">{localize(`${localeKey}.permissions.step`)}</Text>
+    <table>
+        {#each permissionPreferences as permission}
+            <div class="w-full flex flex-row justify-between p-4">
+                <Text>{permission.label}</Text>
+                <Text textColor={permission.required || permission.enabled ? 'success' : 'warning'}>
+                    {localize(`general.${permission.required ? 'required' : permission.enabled ? 'yes' : 'no'}`)}
+                </Text>
+            </div>
+        {/each}
+    </table>
+
+    <Text textColor="secondary">{localize(`${localeKey}.networks.step`)}</Text>
+    <table>
+        {#each networkPreferences as network}
+            <div class="w-full flex flex-row justify-between p-4">
+                <Text>{network}</Text>
+                <Text textColor="success">{localize('general.connected')}</Text>
+            </div>
+        {/each}
+    </table>
+
+    <Text textColor="secondary">{localize(`${localeKey}.accounts.step`)}</Text>
+    <table>
+        {#each accountPreferences as account}
+            <div class="w-full flex flex-row justify-between p-4">
+                <Text>{account.name}</Text>
+                <div />
+            </div>
+        {/each}
+    </table>
+</selection-component>
+
+<style lagn="scss">
```

Comment:
Fix typo and always use postcss. It has better linting support.
```suggestion
<style lang="postcss">
```